### ISSUE TITLE
bug: ERROR in Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by iconio** [ionic-team/ionic#21018]
# Bug Report
My project was working perfectly well before I updated to the latest ionic/cli today, April 11, 2020. Now I can't compile my project. I get the following error:

>ERROR in Cannot read property 'map' of undefined
[ERROR] An error occurred while running subprocess ng.

I've tried using the `ionic cordova emulate ios --livereload --no-native-run --verbose` as well as `ionic cordova build android --verbose` and no further information as for the error.

**Ionic version:**

[x] **6.5.0**

**Ionic info:**

```
```
Ionic:

   Ionic CLI                     : 6.5.0 (/Users/wd/.nvm/versions/node/v12.11.1/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.0.7
   @angular-devkit/build-angular : 0.803.26
   @angular-devkit/schematics    : 8.3.26
   @angular/cli                  : 8.3.26
   @ionic/angular-toolkit        : 2.2.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : not available
   Cordova Plugins   : not available

Utility:

   cordova-res : 0.11.0
   native-run  : 1.0.0

System:

   ios-deploy : 1.10.0
   ios-sim    : 8.0.2
   NodeJS     : v12.11.1 (/Users/wd/.nvm/versions/node/v12.11.1/bin/node)
   npm        : 6.14.4
   OS         : macOS Catalina
   Xcode      : Xcode 11.4 Build version 11E146
```
```

**Comment by NikolaPeevski** [comment]
DId you try to remove the old platforms ?

**Comment by iconio** [comment]
I did. It was quite a stressful time for me. I've removed the platforms and added again and nothing. I ended up doing the following:

1. I ran `ionic cordova plugin save` that "recreated" the plugins list, now I could see them when I ran 'ionic info'.

2. Updated the project to Angular 9 by using `ng update @angular/cli @angular/core`

3. Removed both iOS and Android Platform and recreated them using `ionic platform add ios` and `ionic platform add android`.

Thank God that worked.
